tensorflow_lite_support/cc/task/processor/image_preprocessor.cc

```diff
@@ -114,6 +133,38 @@ absl::Status ImagePreprocessor::Preprocess(const FrameBuffer& frame_buffer,
   } else {
     // Input frame buffer already targets model requirements: skip image
     // preprocessing. For RGB, the data is always stored in a single plane.
+
+    // If dynamic, it will re-dim the entire graph as per the input.
+    if (IsInputImageDynamic()) {
```

Comment:
We can compute IsInputImageDynamic in Init and save it to a bool, same as height_mutable and width_mutable.

Reply:
We would need to create a member to store this state then. Is that okay?

Reply:
Of course. Feel free to add extra member fields.